includes/class-bsr-db.php

```diff
@@ -341,6 +341,7 @@ public function recursive_unserialize_replace( $from = '', $to = '', $data = '',
 					$_tmp = $data;
 					$props = get_object_vars( $data );
 					foreach ( $props as $key => $value ) {
+						$key = trim( $key );
 						$_tmp->$key = $this->recursive_unserialize_replace( $from, $to, $value, false, $case_insensitive );
```

Comment:
I think this would be better, see https://github.com/deliciousbrains/better-search-replace/issues/71#issuecomment-1369195244:

```suggestion
						if (!str_starts_with($key , "\0")) {
							$_tmp->$key = $this->recursive_unserialize_replace( $from, $to, $value, false, $case_insensitive );
						}
```

Reply:
@jdreesen certainly! :) Resolved.